test/token/CodexTitleAccess.test.js

```diff
@@ -0,0 +1,254 @@
+import assertRevert from '../helpers/assertRevert'
+
+const { BigNumber } = web3
+const CodexTitle = artifacts.require('CodexTitle.sol')
+
+require('chai')
+  .use(require('chai-as-promised'))
+  .use(require('chai-bignumber')(BigNumber))
+  .should()
+
+contract('CodexTitleAccess', async function (accounts) {
+  const creator = accounts[0]
+  const another = accounts[1]
+  const firstTokenId = 0
+  const providerId = '1'
+  const providerMetadataId = '10'
+
+  const firstTokenMetadata = {
+    name: 'First token',
+    description: 'This is the first token',
+    imageBytes: 'asdf',
+  }
+
+  const hashedMetadata = {
+    name: web3.sha3(firstTokenMetadata.name),
+    description: web3.sha3(firstTokenMetadata.description),
+    imageBytes: web3.sha3(firstTokenMetadata.imageBytes),
+  }
+
+  beforeEach(async function () {
+    this.token = await CodexTitle.new({ from: creator })
+    await this.token.initializeOwnable(creator)
+
+    await this.token.mint(
+      creator,
+      hashedMetadata.name,
+      hashedMetadata.description,
+      hashedMetadata.imageBytes,
+      providerId,
+      providerMetadataId,
+      { from: creator }
+    )
+  })
+
+  describe('mint', function () {
+    describe('when contract paused', function () {
+      beforeEach(async function () {
+        // Pause the contract
+        await this.token.pause()
+      })
+
+      afterEach(async function () {
+        // unpause the contract
+        await this.token.unpause()
```

Comment:
Why do you need to unpause afterwards? Each test case uses a brand new contract

Reply:
(Meaning that this unpause happens on a contract that won't be used for any other test cases)